### PR TITLE
INGK-660 Catch EoE service deinit error when disconnecting the drive

### PR DESCRIPTION
**Pull request by pzinemanas**
### Description

Please include a summary of the change and which issue is fixed. 

Fixes # INGK-660

### Changes

- [x] Catch deinit error when disconnecting the drive.

### Tests
- [x] Run tests.

Please describe the tests that you ran to verify your changes if it applies. 

### Documentation
- [x] Add the changes at the `[Unreleased]` section of the [CHANGELOG](CHANGELOG.md).

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingenialink tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### CODE REVIEW

File: ingenialink/eoe/network.py [hunk 1]
```diff
@@ -128,7 +128,10 @@ def disconnect_from_slave(self, servo):
         if len(self.servos) == 0:
             self._stop_eoe_service()
             self._erase_config_eoe_service()
-            self._deinitialize_eoe_service()
+            try:
+                self._deinitialize_eoe_service()
```
Comment (GullinBustin) [inline ingenialink/eoe/network.py:132]: Inside the function, don't set `self._eoe_service_init = False` if the command fails

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed
+- Catch EoE service deinit error when disconnecting the drive.
```
Comment (GullinBustin) [inline CHANGELOG.md:6]: It is out of the scope, but add in the CHANGELOG.md virtual drive as a new feature, we forget it in the past

**Review by GullinBustin — CHANGES_REQUESTED** [review]

File: ingenialink/eoe/network.py (hunk 1 above)
Comment (pzinemanas) [reply ingenialink/eoe/network.py:132]: Done!

File: CHANGELOG.md (hunk 2 above)
Comment (pzinemanas) [reply CHANGELOG.md:9]: Done!

**Review by GullinBustin — APPROVED** [review]

**Pull request merged**